test/features/resource.admin.feature

```diff
@@ -146,7 +146,7 @@ Feature: Resource
     And I press "Save"
     Then I should see "Resource Resource 02 edited has been updated"
     When I click "Revisions"
-    Then I should see "current revision"
+    Then I should see "This is the published revision"
```

Comment:
It's just a change on the text displayed on the 'Revisions' section. It was "current revision" before and now it's "This is the published revision".